org.lflang/src/org/lflang/generator/LFGenerator.java

```diff
@@ -132,7 +132,9 @@ private GeneratorBase createGenerator(Target target, FileConfig fileConfig,
             case Python: {
                 return new PythonGenerator(fileConfig, errorReporter);
             }
+            // FIXME: make case exhaustive
```

Comment:
Making the switch exhaustive without default actually didn't work due to `Missing return statement` error.

Please see here for more information: https://stackoverflow.com/questions/39225553/complete-switch-on-enum-in-java-produces-missing-return-statement-error